Cargo.toml

```diff
@@ -51,9 +52,10 @@ odds = { version = "0.4.0" }
 rand = "0.5.5"
 
 [features]
+rayon = ["graphmap", "dep:rayon", "indexmap/rayon"]
```

Comment:
I don't think this should force `graphmap` on -- that happens to be the only useful place for rayon functionality _right now_, but there could be others later.

Reply:
Agreed.